openid-federation-1_0.xml

```diff
@@ -1274,6 +1274,14 @@
 	  IANA "OAuth Dynamic Client Registration Metadata" registry
 	  <xref target="IANA.OAuth.Parameters"/>.
         </t>
+
+        <t>
+          The <spanx style="verb">client_id</spanx> parameter value configured
+          in the <spanx style="verb">openid_relying_party</spanx> metadata MUST
+          be the same of Federation Entity identifier, (
+          <spanx style="verb">iss</spanx> parameter within the Entity Configuration).
+        </t>
+
```

Comment:
This statement is only true when using Automatic Registration.  Therefore it doesn't belong here.  It might be OK to put it in the [Automatic Registration section](https://openid.net/specs/openid-federation-1_0-ID4.html#section-12.1) or one of its subsections.